index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RESUME Roman Levsha</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="name">
+        <div class="container">
+          <div class="name__inf">
+          <h1>Levsha Roman</h1>
+          <p><em>JavaScript beginner programmer</em></</p>
+          </div>
+        </div>
+    </header>
+    <section class="basic">
+        <div class="container">
```

Comment:
Same as above
`<div class="container">`
you can use upper <section> as a container. Avoid using extra-wrappers, please.